.bumpversion.cfg

```diff
@@ -45,3 +45,10 @@ values =
 [bumpversion:file:harness/setup.py]
 
 [bumpversion:file:helm/charts/determined/Chart.yaml]
+
+[bumpversion:file:model-hub/model_hub/__version__.py]
+
+[bumpversion:file:model-hub/setup.py]
+
+[bumpversion:file:model-hub/examples/huggingface/token-classification/ner_config.yaml]
```

Comment:
Is this bump version only for the environment image in that example?

Reply:
removed this.  in its place, we will have to call make -C model_hub update-config-envs-release for releases.